.github/workflows/project.yml

```diff
@@ -18,7 +18,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        go-version: [1.18, 1.19, 1.20.4]
+        go-version: [1.19, 1.20.4]
```

Comment:
the module setup-envtest requires go>=1.19, https://pkg.go.dev/sigs.k8s.io/controller-runtime/tools/setup-envtest